16 colors

eylles opened the pull request:
the heavy work for this PR was done by sonjiku in #601, this PR is a clean up of that one in hopes that the this will be merged now that all the possible reasons as to why #601 wasn't getting merged have been adressed.

Commit:
Made all backend scripts, use the newly modified adjust function from colors.py, that makes pywal generate darker shades for 1-7

Commit:
Fixed wal backend generating only 9 colors

Commit:
Added command line option --nine, which reverts to generating 9 colors, and alacritty output templates.

eylles committed:
remove added templates

these should be their own PR

eylles committed:
Dont separate 9 and 16 cols themes wiht dirs

this separation should be done in other way, not with dirs.

eylles committed:
old adjust for color 0

eylles committed:
remove ref to 9 cols themes

eylles committed:
change --nine to --cols16 restore old behaviour

now the default behaviour is 9 colors again, and 16 colors is a flag

eylles committed:
use 16 color adjust in fast_colorthief

sonjiku commented:
Great work on this @eylles! I really appreciate you mentioning me and adding me as collaborator. I really hope this is going to get merged!

eylles commented:
Welp, @sonjiku your work was the original PR, if it doesn't get merged even after this, what do u say about we collaborate on a more permanent fork? All we need is a name and to add the package to pip.

eylles commented:
welp, i may be a bit too impatient but will start to work on a branch to merge some stuff from PRs and perhaps prepare a repo maybe call it pywal16 ?

eylles commented:
welp, created a detached repo with my branch from this pr as master branch, will start merging some of the open PRs of pywal over there.
https://github.com/eylles/pywal16

eylles committed:
correct name of cached theme json

the names used to be backwards with a 9 being added to 16 color themes and
visceversa.

eylles commented:
somehow the fork still lives https://pypi.org/project/pywal16/
